### PR TITLE
194/update readme

### PR DESCRIPTION
**Pull request by mediumTaj**
### Summary

Update readme files to include Assistant and add VersionDate to instantiation instructions.

### CODE REVIEW

File: src/IBM.WatsonDeveloperCloud.Assistant.v1/README.md [hunk 1]
```diff
@@ -0,0 +1,371 @@
+[![NuGet](https://img.shields.io/badge/nuget-v2.1.0-green.svg?style=flat)](https://www.nuget.org/packages/IBM.WatsonDeveloperCloud.Assistant.v1/)
+
+### Assistant
+
+With the IBM Watson™ [Assistant][assistant] service, you can create an application that understands natural-language input and uses machine learning to respond to customers in a way that simulates a assistant between humans.
```
Comment (lpatino10) [inline src/IBM.WatsonDeveloperCloud.Assistant.v1/README.md]: "a assistant"

File: src/IBM.WatsonDeveloperCloud.Assistant.v1/README.md [hunk 2]
```diff
@@ -0,0 +1,371 @@
...
+  Input = new InputData()
+  {
+    Text = <input-string0>
+  }
+};
+
+//  send a message to the assistant instance
+var result0 = _assistant.Message(<workspace-id>, messageRequest0);
+
+//  reference the message context to continue a assistant
```
Comment (lpatino10) [inline src/IBM.WatsonDeveloperCloud.Assistant.v1/README.md]: Same as the first comment.

**Comment by mediumTaj** [comment]
@lpatino10 Good catch! I revised and also realized I needed to change the import to project file from project.json to .csproj. Added that to each readme. Can you take one more look?

**Comment by mediumTaj** [comment]
Failing because of Vis Rec service - Gonna try later.

**Pull request merged**